quartz/util/fsOps.ts

```diff
@@ -0,0 +1,26 @@
+import { promises as fs, Dirent } from "fs"
+import { join } from "path"
+
+export async function cleanDirectory(dirPath: string): Promise<void> {
+  try {
+    // Check if the directory exists
+    await fs.access(dirPath)
+    const entries: Dirent[] = await fs.readdir(dirPath, { withFileTypes: true })
```

Comment:
`fs.promises.readdir` already returns `ENOENT` if the dir doesn't exist, the `fs.access()` check if redundant